clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -5087,6 +5087,27 @@ static void handleSYCLRegisterNumAttr(Sema &S, Decl *D, const ParsedAttr &AL) {
   D->addAttr(::new (S.Context) SYCLRegisterNumAttr(S.Context, AL, RegNo));
 }
 
+static void handleSYCLIntelESimdWidenAttrAttr(Sema &S, Decl *D,
+                                              const ParsedAttr &Attr) {
```

Comment:
```suggestion
                                              const ParsedAttr &PA) {
```
Or any other identifier that isn't a type name.

Reply:
Done